### PR TITLE
fashion-mnist example

### PR DESCRIPTION
**Pull request by kashif**
Add fashion-mnist helper

### CODE REVIEW

File: mnist/main.py [hunk 1]
```diff
@@ -57,7 +57,9 @@ def main():
     np.random.seed(seed)
 
     # Load the data
-    train_images, train_labels, test_images, test_labels = map(mx.array, mnist.mnist())
+    train_images, train_labels, test_images, test_labels = map(
+        mx.array, mnist.fashion_mnist() if args.fashion_mnist else mnist.mnist()
```
Comment (awni) [inline mnist/main.py:61]: If you make the argument change below, then you can change this to

```suggestion
        mx.array, getattr(mnist, args.dataset),
```

File: mnist/README.md [hunk 2]
```diff
@@ -14,10 +14,16 @@ Run the example with:
...
 python main.py --gpu
 ```
 
 To run the PyTorch or Jax examples install the respective framework.
+
+To run the example with the Fashion-MNIST dataset, use:
+
+```
+python main.py --fashion_mnist
+```
```
Comment (awni) [inline mnist/README.md:29]: Could you just say "For a full list of options run `python main.py --help`" ? We want to keep the README minimal and general for this example.

**Review by awni — CHANGES_REQUESTED** [review]
Thanks for adding this! Just a couple minor comments then we can merge it in!

**Comment by kashif** [comment]
ok sure let me fix it up!

**Review by awni — APPROVED** [review]
Beautiful, thanks!

**Comment by kashif** [comment]
I thank you!

**Pull request merged**